core/rio/api/src/main/java/org/eclipse/rdf4j/rio/RDFFormat.java

```diff
@@ -223,6 +223,17 @@
 			SimpleValueFactory.getInstance().createIRI("http://www.w3.org/ns/formats/JSON-LD"), SUPPORTS_NAMESPACES,
 			SUPPORTS_CONTEXTS, NO_RDF_STAR);
 
+	/**
+	 * The NDJSON-LD</a> is a Newline Delimited JSON-LD format.
```

Comment:
There's a `</a>` here without an opening tag. I assume you wanted to link to the format spec here?

Reply:
It is a bad copy/paste leftover. Unfortunately, there is no format spec, so I will just remove the closing </a> tag. Thanks

Reply:
Done

Reply:
I also added .jsonl and .ndjson extensions in analogy with other parsers, i.e the the rdfxml one. What do you think of also adding .json to to the JSONLD format?

Reply:
I'm not necessarily against it but it's a little out of scope for this, plus we already link the .json extension (and the mime-type) with the SPARQL JSON result format. I'm not 100% sure that using it for two separate formats would not cause problems (e.g. duplicates in the registry). Perhapss best to address this as a separate task so we can do some testing without it holding up this PR.

Reply:
Yes, of course, this should not be part of this MR. I only meant to raise a discussion on it.